### ISSUE TITLE
Update base image for Meteor 1.8 release canditate

### ISSUE THREAD

**Issue by impactmass** [reactioncommerce/reaction#4631]
Tracking reactioncommerce/base#35

- Update Dockerfile after 1.8-rc14 base image is create.
- Use node 8.11.4
- Use meteor npm ci

Update:
npm ci - first clears the node_modules folder. In our case we mounted the node_modules folder. When ci tries to clear the folder, it gets stuck or throws `rm: cannot remove 'node_modules/': Device or resource busy` A workaround is first needed on this, so we can break that out as separate issue.

**Comment by impactmass** [comment]
Currently testing with a test version of the 1.8rc base image: `reactioncommerce/base:v1.8-rc.14-meteor` confirming the new changes works well in 2.0

**Comment by spencern** [comment]
Resolved in #4750

**Comment by impactmass** [comment]
#4750 updates the project code/version to 1.8. 

A second step that we normally do is changing the base image which will provide the meteor tool 1.8 - I made a [PR](https://github.com/reactioncommerce/base/pull/36) for this.

But there's something unclear to me about this Meteor tooling while testing. 
- When I run the current rc-6 branch, because it is still using the v1.7.0.3 of Meteor in it's Dockerfile, I expected that it will attempt to update to v1.8 to match what is specified in the project, but that did not happen. And when I checked the container, it was using the right version

```
$ d exec -it 420cb6548267 bash
node@420cb6548267:/opt/reaction/src$ cd ~/.meteor
node@420cb6548267:~/.meteor$ meteor node --version
v8.11.3
node@420cb6548267:~/.meteor$ meteor npm --version
5.10.0
node@420cb6548267:~/.meteor$ cd /opt/reaction/src/
node@420cb6548267:/opt/reaction/src$ meteor node --version
v8.11.4
node@420cb6548267:/opt/reaction/src$ meteor npm --version
6.4.1
```

So does that mean that specified base version is irrelevant? I don't yet have an answer to that.

**Comment by spencern** [comment]
Yep. you're right, I didn't read this issue clearly enough.

**Comment by spencern** [comment]
I believe that Meteor will update it's version to match whatever the application requires when it gets run (`reaction run` will run `meteor`). My understanding is that we're updating the base branch to eliminate this need and speed up the build process.

**Comment by impactmass** [comment]
That's my understanding too. I just didn't see it try updating, and also didn't notice an apparent delay in the build. But anyway, I'll make a PR in reaction as soon as the base PR is merged.

**Comment by spencern** [comment]
@impactmass can we close this now?

**Comment by impactmass** [comment]
yes, please. Thanks for surfacing. I thought it was closed automatically with this PR #4760
